recipes/PySimpleGUIWeb/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pysimpleguiweb" %}
+{% set version = "0.35.0" %}
+{% set styledName = "PySimpleGUIWeb" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ styledName }}-{{ version }}.tar.gz
+  sha256: "2cb85ec488d3046ba8300a8b6f58a2475e4cff6bc208b8303cec9052df823a70"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.4
+    - remi
```

Comment:
Are you sure `remi` should be in the `host` section and not `run`? Usually only `pip` and `python` belong in the `host` section, and all other dependencies in `run`.